docker/local-node/entrypoint.sh

```diff
@@ -146,8 +146,22 @@ else
     zk contract initialize-validator
   fi
 
-  ## init hyperchain
-  zk contract register-hyperchain
+
+  if [ -z "$CUSTOM_BASE_TOKEN" ]; then
+    echo "Starting chain with ETH"
```

Comment:
nit: might be a confusing log if it doesn't specify it's a gas token (also below)

"Starting chain with ETH as a gas token" instead?

Reply:
done.